06-building-a-program-start-to-finish/writing-tests/output/invertedIndexSpec.js

```diff
@@ -0,0 +1,264 @@
+const uniqueTermsBook = ['alice', 'falls', 'into', 'a', 'rabbit'];
```

Comment:
Is this "uniqueTermsBook" used anywhere in this file? Why declare it if you don't intend to use it.